CHANGELOG.md

```diff
@@ -21,6 +21,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - Refactor `GPUStatsMonitor` to improve training speed ([#3257](https://github.com/PyTorchLightning/pytorch-lightning/pull/3257))
 
+- Changed IoU score behavior for classes not present in target or pred ([#3098](https://github.com/PyTorchLightning/pytorch-lightning/pull/3098))
```

Comment:
Thanks for catching, should have updated there as well.

I think the double-negative in "not present" and "neither/nor" is typically avoided? I think "present in neither target nor pred" or "not present in either target or pred" are both grammatically correct.

Because of all this negating ambiguity and convolution, I actually just went with "absent" instead of "present in neither" or "not present" - "Changed IoU score behavior for classes absent in target and pred" in https://github.com/PyTorchLightning/pytorch-lightning/pull/3098/commits/8b8cc4b8badb188253dcac61a9c3e7a8b9cb7eba . This also matches the updated `absent_score` used in the code that replaced the negation-using `not_present_score`.

Hopefully `absent` is more clear throughout, but please let me know if anyone thinks otherwise.